src/Extensions/LoadingExtension.cs

```diff
@@ -36,36 +34,19 @@ public override void OnLevelLoaded(LoadMode mode)
                 Command.SendToServer(new ClientLevelLoadedCommand());
             }
 
-            UIView uiView = UIView.GetAView();
-
-            // Add the chat log
-            uiView.AddUIComponent(typeof(ChatLogPanel));
-
-            _multiplayerButton = (UIButton)uiView.AddUIComponent(typeof(UIButton));
-
-            _multiplayerButton.text = "Multiplayer";
-            _multiplayerButton.width = 150;
-            _multiplayerButton.height = 40;
-
-            _multiplayerButton.normalBgSprite = "ButtonMenu";
-            _multiplayerButton.disabledBgSprite = "ButtonMenuDisabled";
-            _multiplayerButton.hoveredBgSprite = "ButtonMenuHovered";
-            _multiplayerButton.focusedBgSprite = "ButtonMenuFocused";
-            _multiplayerButton.pressedBgSprite = "ButtonMenuPressed";
-            _multiplayerButton.textColor = new Color32(255, 255, 255, 255);
-            _multiplayerButton.disabledTextColor = new Color32(7, 7, 7, 255);
-            _multiplayerButton.hoveredTextColor = new Color32(7, 132, 255, 255);
-            _multiplayerButton.focusedTextColor = new Color32(255, 255, 255, 255);
-            _multiplayerButton.pressedTextColor = new Color32(30, 30, 44, 255);
-
-            // Enable button sounds.
-            _multiplayerButton.playAudioEvents = true;
+            UIButton resumeButton = UIView.GetAView()?.FindUIComponent("Resume") as UIButton;
```

Comment:
I don't think we should replace the Resume button but rather add another button to the list.

Reply:
Where do think the button should be added instead? Under the resume button?

Reply:
Where do you think the button should be instead? Under it?

Reply:
Yes, I think below would be good.

Reply:
Made two version.
![CSM-pauseMenu](https://user-images.githubusercontent.com/17964845/106237593-6987e300-61ff-11eb-9db4-512a77b2fe5c.jpg)
![thumbnail_CSM-multiplayerBtn-2](https://user-images.githubusercontent.com/17964845/106237598-6c82d380-61ff-11eb-86dc-743eb835f37f.jpg)

Reply:
Seeing this, I think I prefer the first version. What do you think?
And to other people reading this: Do you have any opinion?

Reply:
Yeah I also prefer the first one, it's easier to notice and I think it overall makes more sense.

Reply:
Added the first version